Makefile

```diff
@@ -78,6 +78,7 @@ system-tests: $(PYTHON_BIN) apm-server.test
 	INTEGRATION_TESTS=1 TZ=UTC $(PYTHON_BIN)/nosetests $(NOSETESTS_OPTIONS) $(SYSTEM_TEST_TARGET)
 
 .PHONY: docker-system-tests
+docker-system-tests: export SYSTEM_TEST_TARGET=$(SYSTEM_TEST_TARGET)
```

Comment:
sorry, my mistake: this should be `export SYSTEM_TEST_TARGET:=$(SYSTEM_TEST_TARGET)` (note the colon)

Reply:
I'll update the PR ;)